File: morningsnacks/Lcm.java
import java.util.Scanner;
    public class Lcm{
        public static void main (String[] args){

        Scanner input = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int numberOne = input.nextInt();

        System.out.print("Enter another number: ");
        int numberTwo = input.nextInt();

        int lcm = numberOne;
    
        if (numberTwo > lcm){
            lcm = numberTwo;
            }

        while (true) {
            if (lcm % numberOne == 0 && lcm % numberTwo == 0) {
                break;
                }
            lcm++;    
            }
        System.out.println("Lcm of " + numberOne + " and " + numberTwo + " is " + lcm);

        }
    }

File: morningsnacks/lcm.py
number_one = int(input('Enter a number: '))
number_two = int(input('Enter another number: '))

lcm = number_one

if number_two > lcm:
    lcm = number_two

while True:
    if lcm % number_one == 0 and lcm % number_two == 0:
        break
    lcm += 1
print('LCM of', number_one, "and", number_two, "is", lcm)
